align/cell_fabric/grid.py

```diff
@@ -90,7 +90,6 @@ def __init__( self, *, pitch, offset=0):
 class CenterLineGrid(Grid):
 
     def addCenterLine( self, value, width, isLegal=True, *, color=None):
-        assert width % 2 == 0
```

Comment:
This one we still need. How are you going to draw the outline of the wires on an integer grid if the width is odd.
We'll need to convert the units to half nanometers (or Angstroms).

Reply:
The abstracted grid is of integers. Why does the physical grid need to be of integer values?

Reply:
They are integers in most physical design systems (including ADR.) I think we make all the numbers in the JSON bigger by some factor (either 2 or 10) and scale when we write out GDS. We needed to do this already for ASAP7. @arvuce22 How did we do it for ASAP7? Do we have a number in the JSON file that corresponds to the scale factor we need during GDS generation?